azext_iot/_params.py

```diff
@@ -206,6 +206,22 @@ def load_arguments(self, _):
         context.argument('content_type', options_list=['--content-type', '--ct'],
                          help='MIME Type of file.')
 
+    with self.argument_context('iot device get-parent') as context:
+        context.argument('device_id', help='Id of non-edge device.')
+
+    with self.argument_context('iot device children add') as context:
+        context.argument('device_id', help='Id of edge device.')
+
+    with self.argument_context('iot device children add') as context:
+        context.argument('child_list', options_list=['--child-list'],
+                         help='Child device list includes only non-edge devices.')
+        context.argument('force', options_list=['--force', '-f'], help='Overwrites the non-edge device\'s parent device.')
+
+    with self.argument_context('iot device children remove') as context:
+        context.argument('child_list', options_list=['--child-list'],
```

Comment:
The options list for parameters should include short aliases (like --cl or similar) unless there is a reason the user should spell the whole thing out.

Reply:
done